README.md

```diff
@@ -33,7 +33,8 @@ Note that all user settings (language, keyboard layout, display mode) will persi
 **Raspberry Pi 4 Model B:** display must be connected to HDMI port closest to power jack.
 
 To set up a blank SD card with NOOBS:
-- Format an SD card that is 8GB or greater in size as FAT32 (see instructions on how to do this below)
+- Format an SD card as FAT32 (see instructions on how to do this below). The size of the SD card depends on which operating system(s) you want to install. 
+For a full Raspbian installation, a 16GB card is considered the minimum at the time of writing. To maximise the amount of space available on your SD card, consider using NOOBS-Lite with an internet connecton.
```

Comment:
That sentence is a bit rambly. What about:

"If you wish to use the NOOBS installation, you will need an SD card with a capacity of at least 16GB. We recommend using the NOOBS Lite installation, since this will allow you maximise the amount of space available on the SD card. If you use NOOBS Lite, your Raspberry Pi will need to be connected to the internet to perform the installation.

Reply:
Might be worth just copying the text from https://www.raspberrypi.org/documentation/installation/noobs.md ? Note the capital F in "Full Raspbian".

Reply:
Yes, that would be good for consistency. Although note that the download page doesn't actually have a "Full Raspbian" - they are named:

- Raspbian Buster with desktop and recommended software
- Raspbian Buster with desktop
- Raspbian Buster Lite

Reply:
"For a Raspbian Buster with desktop and recommended software installation, you will need an SD card with a capacity of at least 16GB." is a bit of a mouthful?? :laughing:

Reply:
@Lurch - Yes, I think the text was once the same, but the website seems to have been improved and so has now diverged. So I think just updating this file to match the website is best as indicated in #581. So shall just close this PR?

Reply:
Yes, that would be a mouthful. I was not proposing that as a solution, just pointing out the inconsistency. My proposal would be three options:

1. Raspbian Full
2. Raspbian Medium
3. Raspbian Lite

with some explanation on the download page as to what each one is. That keeps the names short and consistent, while also providing some useful guidance for folk.

Reply:
I think this PR (with tweaks) is fine, as we want the NOOBS readme to match what https://www.raspberrypi.org/documentation/installation/noobs.md says. Having one place recommending a 16GB card but another place recommending an 8GB card is only going to cause confusion :slightly_smiling_face:

Reply:
connection is spelt wrongly. But otherwise seems fine?

Reply:
@JamesH65 wins a gold star for spotting a spelling mistake that I missed :rofl: 
:star:

Reply:
I'm currently formulating a cunning plan to run the whole repo through a spell checker. If anyone knows a good UK English spell checker for Linux, let me know.

Reply:
Surely there's not _that_ many typos in NOOBS? :confused: 

I've not looked at in detail, but a previous project I worked on used https://github.com/codespell-project/codespell

Reply:
I was meaning the documentation repo. Forgot this one was in the NOOBS repo. Actually it would be handy to run it on all repos, although that would probably just generate a lot of unnecessary work for people. Running a spell checker on the documentation repo would be useful though.